.github/workflows/e2e_user_features.yml

```diff
@@ -0,0 +1,53 @@
+name: End-to-End tests
+on:
+  push:
+    branches:
+      - develop
+      - release-*
+  pull_request:
+    branches:
+      - develop
+      - release-*
+
+jobs:
+  e2e_additional_editor_and_player:
+    runs-on:  ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '2.x'
+          architecture: 'x64'
+      - name: Setup python by installing wheel
+        if: startsWith(github.head_ref, 'update-changelog-for-release') == false
+        run: pip install wheel==0.35.0
+      - name: Install third party
+        if: startsWith(github.head_ref, 'update-changelog-for-release') == false
+        run: python -m scripts.install_third_party_libs
+      - name: Install chrome version 88.0.4324.96-1
+        if: startsWith(github.head_ref, 'update-changelog-for-release') == false
+        run: |
+          sudo apt-get update
+          sudo apt-get install libappindicator3-1
+          curl -L -o google-chrome.deb https://github.com/webnicer/chrome-downloads/raw/master/x64.deb/google-chrome-stable_88.0.4324.96-1_amd64.deb
+          sudo sed -i 's|HERE/chrome\"|HERE/chrome\" --disable-setuid-sandbox|g' /opt/google/chrome/google-chrome
+          sudo dpkg -i google-chrome.deb
+          google-chrome --version
+      - name: Run e2e preferences test
+        if: startsWith(github.head_ref, 'update-changelog-for-release') == false
+        run: xvfb-run -a python -m scripts.run_e2e_tests --skip-install --skip-build --suite="preferences" --prod_env
```

Comment:
Should the first test in this group avoid the `skip_build` flag like we have done in the other config files?

see example
https://github.com/oppia/oppia/blob/988f7c1e818faf774ec424e33b5dd0267c40237b/.github/workflows/e2e_translation_classroom_and_core_features.yml#L41